### DISCUSSION TITLE
Welcome to omicser Discussions!

### DISCUSSION THREAD

**Discussion by ergonyc** [ndcn/omicser#144] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. For now thats primarily our "brain trust" participants, but others feel free to get involved! We hope that you:
  * Pose questions you’re wondering about to the group.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ergonyc** [comment]
Who is out there?  Please let us know if you are watching/reading!

**Reply by MenonLab** [reply]
Saying hello. Always excited to come here and see the updates.

**Comment by k8hertweck** [comment]
Hi folks! I'm Kate, a member of the CZI Open Science team currently responsible for helping grantees adopt and implement open science approaches. I'm particularly passionate about open source scientific software development and supporting the communities around such work, hence my interest in the omicser project. Looking forward to connecting more and facilitating further development!
